deck/tests/test_functional.py

```diff
@@ -258,6 +258,24 @@ def test_event_create_event_proposal_with_passed_due_date(self):
             )
         self.assertEquals(0, event.proposals.count())
 
+        response = self.client.get(
+            reverse('create_event_proposal', kwargs={'slug': event.slug}))
+        self.assertEqual(302, response.status_code)
+
+    def test_export_event_votes_to_csv(self):
+        event = Event.objects.create(**self.event_data)
+        response = self.client.get(
+            reverse('export_event', kwargs={'slug': event.slug}))
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response['Content-Type'], 'text/csv')
```

Comment:
@mauricioabreu i was looking at the library that we use and i saw a test folder: https://github.com/azavea/django-queryset-csv/tree/master/test_app/djqscsv_tests/tests

Maybe we can found something useful in there.

o/